python/paddle/fluid/tests/unittests/test_case.py

```diff
@@ -223,5 +224,52 @@ def type_error_default():
             self.assertRaises(TypeError, type_error_default)
 
 
+# when optimizer in case
+class TestMutiTask(unittest.TestCase):
+    def test_optimizer_in_case(self):
+        BATCH_SIZE = 1
+        INPUT_SIZE = 784
+        EPOCH_NUM = 2
+
+        x = fluid.data(
+            name='x', shape=[BATCH_SIZE, INPUT_SIZE], dtype='float32')
+        y = fluid.data(
+            name='y', shape=[BATCH_SIZE, INPUT_SIZE], dtype='float32')
+
+        switch_id = fluid.data(name='switch_id', shape=[1], dtype='int32')
+
+        one = layers.fill_constant(shape=[1], dtype='int32', value=0)
```

Comment:
Variable name is "one" but the value is 0. Please rename the variable or set other value

Reply:
Thanks. Done